genesyscloud/tfexporter/resource_genesyscloud_tf_export_test.go

```diff
@@ -7,7 +7,31 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	gcloud "github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud"
+	"io"
+	"log"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"sync"
+	architectFlow "terraform-provider-genesyscloud/genesyscloud/architect_flow"
+	userPromptResource "terraform-provider-genesyscloud/genesyscloud/architect_user_prompt"
+	authDivision "terraform-provider-genesyscloud/genesyscloud/auth_division"
+	obContactList "terraform-provider-genesyscloud/genesyscloud/outbound_contact_list"
+	"terraform-provider-genesyscloud/genesyscloud/platform"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	qualityFormsEvaluation "terraform-provider-genesyscloud/genesyscloud/quality_forms_evaluation"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	routingQueue "terraform-provider-genesyscloud/genesyscloud/routing_queue"
+	routingWrapupcode "terraform-provider-genesyscloud/genesyscloud/routing_wrapupcode"
+	telephonyProvidersEdgesSite "terraform-provider-genesyscloud/genesyscloud/telephony_providers_edges_site"
+	"terraform-provider-genesyscloud/genesyscloud/user"
+	"terraform-provider-genesyscloud/genesyscloud/util"
```

Comment:
These imports are invalid. We recently renamed the go module for this project from `terraform-provider-genesyscloud` to `github.com/mypurecloud/terraform-provider-genesyscloud` , so all importations of local packages have to be refactored to reflect this. If you run `go vet ./...` it will highlight these instances. Once this command no longer producing anything, you're good to go